.github/workflows/test-evm.yaml

```diff
@@ -131,12 +128,27 @@ jobs:
         run: node evm/interchainTokenFactory.js --action deployInterchainToken --name "test" --symbol "TST" --decimals 18 --minter 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 --salt "salt" -y
 
       - name: InterchainTokenFactory deploy interchain token to destination chain
-        run: node evm/interchainTokenFactory.js --action deployRemoteInterchainToken --destinationChain remote --minter 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 --salt "salt" -y
+        run: node evm/interchainTokenFactory.js --action deployRemoteInterchainToken --destinationChain remote --salt "salt" -y
+
+      # Note that tokenAddress is hardcoded since it's derivation must always be the same
```

Comment:
```suggestion
      # Note that tokenAddress is hardcoded since it's derivation must always be the same
      - name: InterchainTokenService register token metadata
         run: node evm/its.js --action registerTokenMetadata --tokenAddress 0x49c06259B42540a025A73a32eF2Fd183c0FDB1D2 -y

```